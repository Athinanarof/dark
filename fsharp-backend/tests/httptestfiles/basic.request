GET / HTTP/1.1
Host: basic.builtwithdark.localhost:9001
Date: Sun, 08 Nov 2020 15:38:01 GMT
Content-Length: 0

