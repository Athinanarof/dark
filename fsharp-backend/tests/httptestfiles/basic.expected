HTTP/1.1 200 OK
Date: XXX, XX XXX XXXX XX:XX:XX XXX
Content-Type: application/json; charset=utf-8
Server: darklang
Content-Length: 12

"hello world"